envi/memory.py

```diff
@@ -720,3 +755,45 @@ def memdiff(bytes1, bytes2):
             ret.append((beginoff, offset-beginoff))
         offset += 1
     return ret
+
```

Comment:
while this looks very much like the `collapseMemoryMaps()` in `vtrace.utils`, they differ in one distinctive way:
this deals with envi.memory memorymaps.  
`vtrace.utils.collapseMemoryMaps()` deals with Vtrace memory maps, which are not exactly the same.

Reply:
should i create one function that handles both possibilities?  or change function names?